src/sysl/importers/import_swagger.py

```diff
@@ -109,6 +114,7 @@ def parse_typespec(tspec):
 
     # skip invalid arrays
     if not typ and 'items' in tspec:
+        warn('Ignoring unexpected "items". Schema has "items" but did not have defined "type". Note: %r' % (tspec, ))
```

Comment:
No need to force a tuple for a single argument to the formatting operator. just `% tspec` will do.

Reply:
it will fail in cases where `tspec` is a tuple value with length != 1